Source/core/Portability.h

```diff
@@ -706,7 +706,8 @@ namespace Core {
         ERROR_CODE(ERROR_UNAUTHENTICATED, 42) \
         ERROR_CODE(ERROR_NOT_EXIST, 43) \
         ERROR_CODE(ERROR_NOT_SUPPORTED, 44) \
-        ERROR_CODE(ERROR_INVALID_RANGE, 45)
+        ERROR_CODE(ERROR_INVALID_RANGE, 45) \
+        ERROR_CODE(ERROR_PLUGIN_NOTACTIVATED, 46)
```

Comment:
Mixing domain logic from higher layers  (WPEFramework knowledge) in lower layers is not according to the Thunder architecture. So I think we can remove this error, and as inidcated use the UNAVAILABLE.